app/controllers/App.scala

```diff
@@ -20,9 +24,13 @@ object App extends Controller with PanDomainAuthActions {
       case None => routes.Assets.versioned(jsFileName).toString
     }
 
-    val clientConfig = ClientConfig(Config().capiUrl, Config().capiKey, TagType.list.map(_.name))
+    Permissions.getPermissionsForUser(req.user.email).map { permissions =>
```

Comment:
Apologies if this is a stupid question because I don't know the semantics of the Permissions library, but what happens to this is the permissions service is down?